src/components/application_manager/src/commands/mobile/system_request.cc

```diff
@@ -85,7 +85,7 @@ class QueryAppsDataValidator {
     if (!HasResponseKey()) {
       return false;
     }
-    return ValidateAppDataAndOsAndLanguagesData();
+    return true;
```

Comment:
@AGritsevich Why did you replace big function with lots of logic with one const bool value ?